src/components/EvModuleList.vue

```diff
@@ -72,6 +87,10 @@ export default Vue.extend({
       return Object.entries(configs).map(([key]) => key);
     },
   },
+  data: (): Data => ({
+    showDialog: false,
+    config_to_load: null,
```

Comment:
Naming seems to be inconsistent, some methods use camelCase, some snail_case, same for data. Maybe we could  add a stricter linter-rule?

Reply:
This could be a good idea